examples/experimental/dagster-airlift/dagster_airlift/core/airflow_instance.py

```diff
@@ -85,19 +78,6 @@ def list_variables(self) -> List[Dict[str, Any]]:
                 "Failed to fetch variables. Status code: {response.status_code}, Message: {response.text}"
             )
 
-    def get_proxied_state(self) -> AirflowProxiedState:
-        local_migration_dir = get_local_proxied_state_dir()
-        if local_migration_dir is not None:
-            return load_proxied_state_from_yaml(local_migration_dir)
-        variables = self.list_variables()
```

Comment:
I'm keeping list_variables around in case we need it again.

Reply:
seems reasonable